druid/ingestion-spec/ml_observation_status_index.json

```diff
@@ -0,0 +1,197 @@
+{
+  "type": "index",
+  "spec": {
+    "ioConfig": {
+      "type": "index",
+      "inputSource": {
+        "type": "azure",
+        "uris": [
+          "azure://<container_name>/observation/status/sl_observation_status.json"
```

Comment:
batch ingestion file path is also incorrectly configured.

Reply:
can you please share me the sample ingestion spec which is correct ? , because i go throughed the already existing ingestion spec of diksha.